### ISSUE TITLE
Migrer ce manuel vers https://github.com/bombmanual

### ISSUE THREAD

**Issue by tatref** [gplaurin/keep-talking-and-nobody-explodes-manuel-francais#2]
Salut,

Tout d'abord, merci pour ton travail !

L'organisation bombmanual contient actuellement le manuel en plusieurs langues : anglais, italien, espagnol, allemand, russe.
Les manuels sont ensuite disponibles ici : https://bombmanual.github.io/

Serait-il intéressant d'y ajouter ta traduction ?
Il faudrait demander aux responsables de l'organisation de t'ajouter dans l'organisation pour conserver les droits de commit sur le dépôt.

Qu'en penses-tu ?

**Comment by nagromc** [comment]
Good idea. Otherwise, we could just ask @bombmanual to put on https://bombmanual.github.io a link to the [generated PDF](https://raw.githubusercontent.com/gplaurin/keep-talking-and-nobody-explodes-manuel-francais/master/Manuel-du-d%C3%A9mineur_1.pdf).

**Comment by DavidBabel** [comment]
yep
